cli/run/batchWarnings.ts

```diff
@@ -7,18 +7,25 @@ export interface BatchWarnings {
 	add: (warning: RollupWarning) => void;
 	readonly count: number;
 	flush: () => void;
+	readonly total: number;
 }
 
 export default function batchWarnings() {
 	let deferredWarnings = new Map<keyof typeof deferredHandlers, RollupWarning[]>();
+	let total = 0;
 	let count = 0;
 
 	return {
+		get total() {
+			return total;
+		},
+
 		get count() {
 			return count;
 		},
 
 		add: (warning: RollupWarning) => {
+			total += 1;
```

Comment:
To me it looks like `total` is absolutely identical to `count`?

Reply:
The problem was `count` is reset during the `flush()`. We need to know if a warning occurred at any point, even if they've already been flushed.

Reply:
Ok, then how about not counting at all but just setting a boolean from `false` to `true`?

Reply:
sure will update